packages/matrix-identity-server/src/db/index.ts

```diff
@@ -428,7 +428,7 @@ class IdentityServerDb<T extends string = never>
     )
   }
 
-  //eslint-disable-next-line @typescript-eslint/explicit-function-return-type, @typescript-eslint/promise-function-async
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type, @typescript-eslint/promise-function-async
```

Comment:
we can remove those eslint comments and fix the warning error by providing the return ( `GetMinMaxJoin2<T>` ) value of the function .